File: DS&DAA-using-c/selectionsort.c
#include<stdio.h>

void main(){
    int arr[50],n,i,j,temp,imin;
    /*Taking user input any array*/
    printf("Enter the no of element in array\n");
    scanf("%d",&n);
    printf("Enter the element of array\n");
    for(i=0;i<n;i++){
        scanf("%d",&arr[i]);       
    }
    
    /*Printing unshorted array*/
    printf("Unshorted array\n");
    for(i=0;i<n;i++){
        printf("%d\n",arr[i]);
    }

    /*main stetment where shorting perform */
    for(i=0;i<n-1;i++){
        imin = i;
        for(j=i+1;j<n;j++){
            if(arr[j]<arr[imin]){
                imin = j;
            }
        }
        temp = arr[i];
        arr[i] = arr[imin];
        arr[imin] = temp;
    }

    /*Printing shorted array*/
    printf("Shorted array\n");
    for(i=0;i<n;i++){
        printf("%d\n",arr[i]);
    }

}  

File: DS&DAA-using-python/selectionsort.py
#taking array input from user
n = int(input("Enter the no of ekement in unsoterd array\n"))
arr = []
print("Enter the elements of array")
for i in range(n):
    arr.append(int(input()))

#printing unsorted array
print("Unsorted array\n",arr)

#main stetment where shorting perform
for i in range(0,n-1):
    imin = i
    for j in range(i+1,n):
        if(arr[j]<arr[imin]):
            imin = j
    temp = arr[i]
    arr[i] = arr[imin]
    arr[imin] = temp

#printing unsorted array
print("Sorted array\n",arr)    
